digital-twins/Samples/readme.md

```diff
@@ -0,0 +1,26 @@
+# Azure IoT Digital Twins Samples
+
+This folder contains all the samples for the Azure Digital Twins Public Preview. They are organized in 3 subfolders:
+
+**device** contains all the samples for the `azure-iot-digitaltwins-device` package to create an Azure IoT Plug-and-Play device client:
+
+- Implement interfaces and instantiate them as instances for this device
+- register this device, with its capability model and instances using the `DigitalTwinClient` class.
+- send telemetry, act on commands, receive writable property updates and report property changes.
```

Comment:
- Implement interfaces and instantiate them as instances for this device.
- Register this device with its capability model and instances using the `DigitalTwinClient` class.
- Send telemetry, act on commands, receive writable property updates, and report property changes.